packages/manager/apps/web/client/app/private-database/task/private-database-task.html

```diff
@@ -2,9 +2,20 @@
     <div data-ovh-alert="{{alerts.main}}"></div>
 
     <oui-datagrid
+        id="{{:: taskCtrl.datagridId }}"
         data-rows="taskCtrl.taskDetails"
         data-row-loader="taskCtrl.transformItem($row)"
     >
+        <oui-datagrid-topbar>
+            <oui-button
+                type="button"
+                class="float-right"
+                data-on-click="taskCtrl.getTasks()"
```

Comment:
Why you didn't made as other datagrid with     
refreshTable() {
      return this.ouiDatagridService.refresh(this.datagridId, true);
 }
 
 Because you add a ID on datagrib but you don't use it.

Reply:
Hi,

Because it did not work when I tested on friday, but I tested right now, and seems ok, So I will make this homogenous here